make/image.mk

```diff
@@ -0,0 +1,5 @@
+.PHONY: image
+## build docker image
+image:
+	docker build -t kleinhenz/registration-service:0.1 .
```

Comment:
why do you want to use your username as the org?

Reply:
For now as we don't have a repository yet. We will need more time to find the final location.